### PR TITLE
Trim TOC entries for Remote debug wv2 WinUI 2 (UWP) apps

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered TOC for review: 
https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/remote-debugging?branch=pr-en-us-3161

AB#50912158

### CODE REVIEW

File: microsoft-edge/webview2/how-to/remote-debugging-xbox.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ ms.subservice: webview
 ms.date: 10/17/2023
 ---
 # Remote debugging Xbox WebView2 WinUI 2 (UWP) apps
+<!-- todo: description: YAML field: remove "by using Remote Tools for Microsoft Edge"? -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/remote-debugging-xbox.md:12]: in the description: YAML field, remove "by using Remote Tools for Microsoft Edge"?

rendered:
https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/remote-debugging-xbox?branch=pr-en-us-3161

@champnic

File: microsoft-edge/toc.yml [hunk 2]
```diff
@@ -1414,17 +1414,17 @@
                   href: webview2/how-to/remote-debugging.md
```
Comment (champnic) [inline microsoft-edge/toc.yml:1414]: To consider: I think we could remove this page. All the content is already in the other pages, and TOC is already on the left side navigation.

**Review by champnic — APPROVED** [review]
This looks good as-is, but I also think we could remove the "remote-debugging" TOC page.

File: microsoft-edge/toc.yml (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/toc.yml:1414]: keeping nav page, b/c several articles link to it as a platform-neutral link destination.

File: microsoft-edge/webview2/how-to/remote-debugging-xbox.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/remote-debugging-xbox.md]: removed

**Pull request merged**